### ISSUE TITLE
CPU版本的paddle，使用官方StructureV3模型推理代码不出结果

### ISSUE THREAD

**Issue by 7taozhou7** [PaddlePaddle/PaddleOCR#15780]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

这是运行的log：
INFO: Could not find files for the given pattern(s).
C:\Users\intel\.conda\envs\ocr\lib\site-packages\paddle\utils\cpp_extension\extension_utils.py:711: UserWarning: No ccache found. Please be aware that recompiling all source files may be required. You can download and install ccache from: https://github.com/ccache/ccache/blob/master/doc/INSTALL.md
  warnings.warn(warning_message)
Creating model: ('PP-DocBlockLayout', None)
Using official model (PP-DocBlockLayout), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Creating model: ('PP-DocLayout_plus-L', None)
Using official model (PP-DocLayout_plus-L), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Creating model: ('PP-LCNet_x1_0_textline_ori', None)
Using official model (PP-LCNet_x1_0_textline_ori), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Creating model: ('PP-OCRv5_server_det', None)
Using official model (PP-OCRv5_server_det), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Creating model: ('PP-OCRv5_server_rec', None)
Using official model (PP-OCRv5_server_rec), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Creating model: ('PP-OCRv4_server_seal_det', None)
Using official model (PP-OCRv4_server_seal_det), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Creating model: ('PP-OCRv5_server_rec', None)
Using official model (PP-OCRv5_server_rec), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Creating model: ('PP-LCNet_x1_0_table_cls', None)
Using official model (PP-LCNet_x1_0_table_cls), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Creating model: ('SLANeXt_wired', None)
Using official model (SLANeXt_wired), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Creating model: ('SLANet_plus', None)
Using official model (SLANet_plus), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Creating model: ('RT-DETR-L_wired_table_cell_det', None)
Using official model (RT-DETR-L_wired_table_cell_det), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Creating model: ('RT-DETR-L_wireless_table_cell_det', None)
Using official model (RT-DETR-L_wireless_table_cell_det), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Creating model: ('PP-FormulaNet_plus-L', None)
Using official model (PP-FormulaNet_plus-L), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Creating model: ('PP-Chart2Table', None)
Using official model (PP-Chart2Table), the model files will be automatically downloaded and saved in C:\Users\intel\.paddlex\official_models.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Loading configuration file C:\Users\intel\.paddlex\official_models\PP-Chart2Table\config.json
Loading weights file C:\Users\intel\.paddlex\official_models\PP-Chart2Table\model_state.pdparams
Loaded weights file from disk, setting weights to model.
All model checkpoint weights were used when initializing PPChart2TableInference.

All the weights of PPChart2TableInference were initialized from the model checkpoint at C:\Users\intel\.paddlex\official_models\PP-Chart2Table.
If your task is similar to the task the model of the checkpoint was trained on, you can already use PPChart2TableInference for predictions without further training.
Loading configuration file C:\Users\intel\.paddlex\official_models\PP-Chart2Table\generation_config.json

这是我的环境：
Package                  Version
------------------------ -----------
aiohappyeyeballs         2.6.1
aiohttp                  3.12.13
aiosignal                1.3.2
annotated-types          0.7.0
anyio                    4.9.0
astor                    0.8.1
async-timeout            4.0.3
attrs                    25.3.0
cachetools               6.1.0
certifi                  2025.1.31
chardet                  5.2.0
charset-normalizer       3.4.2
colorama                 0.4.6
colorlog                 6.9.0
conda-pack               0.8.1
cssselect                1.3.0
cssutils                 2.11.1
dataclasses-json         0.6.7
decorator                5.2.1
distro                   1.9.0
einops                   0.8.1
et_xmlfile               2.0.0
exceptiongroup           1.2.2
filelock                 3.18.0
frozenlist               1.7.0
fsspec                   2025.5.1
ftfy                     6.3.1
GPUtil                   1.4.0
greenlet                 3.2.3
h11                      0.14.0
httpcore                 1.0.8
httpx                    0.28.1
httpx-sse                0.4.0
huggingface-hub          0.33.0
idna                     3.10
imagesize                1.4.1
Jinja2                   3.1.6
jiter                    0.10.0
joblib                   1.5.1
jsonpatch                1.33
jsonpointer              3.0.0
langchain                0.3.25
langchain-community      0.3.25
langchain-core           0.3.65
langchain-openai         0.3.24
langchain-text-splitters 0.3.8
langsmith                0.3.45
lxml                     5.4.0
MarkupSafe               3.0.2
marshmallow              3.26.1
more-itertools           10.7.0
multidict                6.5.0
mypy_extensions          1.1.0
networkx                 3.4.2
numpy                    2.2.4
openai                   1.88.0
opencv-contrib-python    4.10.0.84
openpyxl                 3.1.5
opt-einsum               3.3.0
orjson                   3.10.18
packaging                24.2
paddleocr                3.0.2
paddlepaddle             3.0.0
paddlex                  3.0.2
pandas                   2.3.0
pillow                   11.1.0
pip                      25.1.1
premailer                3.10.0
prettytable              3.16.0
propcache                0.3.2
protobuf                 6.30.2
py-cpuinfo               9.0.0
pyclipper                1.3.0.post6
pydantic                 2.11.7
pydantic_core            2.33.2
pydantic-settings        2.9.1
pypdfium2                4.30.1
python-dateutil          2.9.0.post0
python-dotenv            1.1.0
pytz                     2025.2
PyYAML                   6.0.2
regex                    2024.11.6
requests                 2.32.4
requests-toolbelt        1.0.0
ruamel.yaml              0.18.14
ruamel.yaml.clib         0.2.12
scikit-learn             1.7.0
scipy                    1.15.3
setuptools               80.9.0
shapely                  2.1.1
six                      1.17.0
sniffio                  1.3.1
SQLAlchemy               2.0.41
tenacity                 9.1.2
threadpoolctl            3.6.0
tiktoken                 0.9.0
tokenizers               0.21.1
tqdm                     4.67.1
typing_extensions        4.13.2
typing-inspect           0.9.0
typing-inspection        0.4.1
tzdata                   2025.2
ujson                    5.10.0
urllib3                  2.5.0
wcwidth                  0.2.13
wheel                    0.45.1
yarl                     1.20.1
zstandard                0.23.0

这是运行代码：
from pathlib import Path
from paddleocr import PPStructureV3

pipeline = PPStructureV3(
    device='cpu',
    use_doc_orientation_classify=False,
    use_doc_unwarping=False
)

# For Image
output = pipeline.predict(
    input="test_data/structure_data/pp_structure_v3_demo.png",
    )

# 可视化结果并保存 json 结果
for res in output:
    res.print() 
    res.save_to_json(save_path="output_structure") 
    res.save_to_markdown(save_path="output") 

这是输出截图：

![Image](https://github.com/user-attachments/assets/a759c78e-3370-48dc-b478-eb0d2b90334d)

### 🏃‍♂️ Environment (运行环境)

这是我的环境：
Package                  Version
------------------------ -----------
aiohappyeyeballs         2.6.1
aiohttp                  3.12.13
aiosignal                1.3.2
annotated-types          0.7.0
anyio                    4.9.0
astor                    0.8.1
async-timeout            4.0.3
attrs                    25.3.0
cachetools               6.1.0
certifi                  2025.1.31
chardet                  5.2.0
charset-normalizer       3.4.2
colorama                 0.4.6
colorlog                 6.9.0
conda-pack               0.8.1
cssselect                1.3.0
cssutils                 2.11.1
dataclasses-json         0.6.7
decorator                5.2.1
distro                   1.9.0
einops                   0.8.1
et_xmlfile               2.0.0
exceptiongroup           1.2.2
filelock                 3.18.0
frozenlist               1.7.0
fsspec                   2025.5.1
ftfy                     6.3.1
GPUtil                   1.4.0
greenlet                 3.2.3
h11                      0.14.0
httpcore                 1.0.8
httpx                    0.28.1
httpx-sse                0.4.0
huggingface-hub          0.33.0
idna                     3.10
imagesize                1.4.1
Jinja2                   3.1.6
jiter                    0.10.0
joblib                   1.5.1
jsonpatch                1.33
jsonpointer              3.0.0
langchain                0.3.25
langchain-community      0.3.25
langchain-core           0.3.65
langchain-openai         0.3.24
langchain-text-splitters 0.3.8
langsmith                0.3.45
lxml                     5.4.0
MarkupSafe               3.0.2
marshmallow              3.26.1
more-itertools           10.7.0
multidict                6.5.0
mypy_extensions          1.1.0
networkx                 3.4.2
numpy                    2.2.4
openai                   1.88.0
opencv-contrib-python    4.10.0.84
openpyxl                 3.1.5
opt-einsum               3.3.0
orjson                   3.10.18
packaging                24.2
paddleocr                3.0.2
paddlepaddle             3.0.0
paddlex                  3.0.2
pandas                   2.3.0
pillow                   11.1.0
pip                      25.1.1
premailer                3.10.0
prettytable              3.16.0
propcache                0.3.2
protobuf                 6.30.2
py-cpuinfo               9.0.0
pyclipper                1.3.0.post6
pydantic                 2.11.7
pydantic_core            2.33.2
pydantic-settings        2.9.1
pypdfium2                4.30.1
python-dateutil          2.9.0.post0
python-dotenv            1.1.0
pytz                     2025.2
PyYAML                   6.0.2
regex                    2024.11.6
requests                 2.32.4
requests-toolbelt        1.0.0
ruamel.yaml              0.18.14
ruamel.yaml.clib         0.2.12
scikit-learn             1.7.0
scipy                    1.15.3
setuptools               80.9.0
shapely                  2.1.1
six                      1.17.0
sniffio                  1.3.1
SQLAlchemy               2.0.41
tenacity                 9.1.2
threadpoolctl            3.6.0
tiktoken                 0.9.0
tokenizers               0.21.1
tqdm                     4.67.1
typing_extensions        4.13.2
typing-inspect           0.9.0
typing-inspection        0.4.1
tzdata                   2025.2
ujson                    5.10.0
urllib3                  2.5.0
wcwidth                  0.2.13
wheel                    0.45.1
yarl                     1.20.1
zstandard                0.23.0

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

windows11下CPU版本的paddle，使用官方StructureV3模型推理代码不出结果

**Comment by Bobholamovic** [comment]
麻烦看看这个issue里提到的方案有帮助不：

https://github.com/PaddlePaddle/PaddleOCR/pull/15790

**Comment by pxyfxun** [comment]
您好，我也遇到了，请问您的问题解决了吗

**Comment by 7taozhou7** [comment]
> 您好，我也遇到了，请问您的问题解决了吗

还没有解决~

**Comment by pxyfxun** [comment]
> > 您好，我也遇到了，请问您的问题解决了吗
> 
> 还没有解决~

哎。。。

**Comment by Bobholamovic** [comment]
如果是观察到“程序中断没有任何提示”的情况，有可能是因为资源使用（例如内存用量）超过限制，导致进程被系统杀死了。这种情况下，建议可以尝试更换模型，例如使用mobile版本的OCR模型，以及更小尺寸的公式识别模型等。默认的模型是高精度方案，比较吃资源～

**Comment by TingquanGao** [comment]
The issue has no response for a long time and will be closed. You can reopen or new another issue if are still confused.

---
_From Bot_
